applications/swimming_DEM_application/custom_utilities/fields/pouliot_flow_field_2D.cpp

```diff
@@ -61,11 +66,17 @@ double PouliotFlowField2D::U0DT(const int i)
 }
 double PouliotFlowField2D::U0D0(const int i)
 {
-    return - 25 * mExpX[i];
+    const double x = mX[i];
+    KRATOS_WATCH(i)
+    KRATOS_WATCH(x)
+    return 3*x*x;
 }
 double PouliotFlowField2D::U0D1(const int i)
 {
-    return - 25 * mExpY[i];
+    const double y = mY[i];
+    KRATOS_WATCH(i)
```

Comment:
Not sure if Kratos_watch would slow the code, but I would get rid of them (at least in release compilation)